idb/postgres/postgres.go

```diff
@@ -1628,7 +1657,7 @@ func (db *IndexerDb) txnsWithNext(ctx context.Context, tf idb.TransactionFilter,
 		close(out)
 		return
 	}
-	rows, err := db.db.QueryContext(ctx, query, whereArgs...)
+	rows, err := tx.Query(query, whereArgs...)
```

Comment:
why lose the context?

Reply:
I passed the context when I created `tx`. Isn't it used when I query?

Reply:
oh yep, that's right